### DISCUSSION TITLE
Make "High Gas Fees - Block Bucks #2201" the highest priority in Sunflowerland's backlog

### DISCUSSION THREAD

**Discussion by poteitouweb3** [sunflower-land/sunflower-land#2206] [Polls]
As you all know, in the last few weeks the polygon network has been devastated by a game and the congestion is affecting us like never before. Immediately the community and Adam started discussions about proposals for the problem, with the best solution being proposed by adam in the thread **High Gas Fees - Block Bucks #2201**. 

Despite the discussions, the item is still not in the Feature Development backlog (https://github.com/orgs/sunflower-land/projects/24) and we haven't had an official position about the priority or urgency of this feature. So what I'd like to propose is a simple vote, to put as top priority or not - in the eyes of the community - the coming of Block Bucks.

We are a big project with a lot to move forward but all our "eggs" are in one "basket". That is if polygon has problems so does the whole project and we need to seek more resilience than ever before.

**Comment by NycksonV** [comment]
Please Devs apply a solution to this problem not only me but many farmers are getting farm delayed due to the absurd high gas of the polygon, mainly now due to the new event of the treasure island I am stuck most of the day because of the gas that does not low ;-;

**Comment by SuperSynapse** [comment]
I really liked Adam's idea of buying "tickets". For instance 10 tickets for $1 That would prologue needing actual syncs, and instead do a server-side sync instead.

High gas is definitely an issue right now within the community.

I'd also say as the land has expanded, and more items/crops/foods/etc, the contract has gotten a bit more bloated, and adding this to alleviate sync pressure would be greatly appreciated.

**Reply by obesbash** [reply]
> I really liked Adam's idea of buying "tickets". For instance 10 tickets for $1 That would prologue needing actual syncs, and instead do a server-side sync instead.

But keep in mind - this solution will decrease resource cost at inner market SFL.

**Comment by Xeko-92** [comment]
>Despite the discussions, the item is still not in the Feature Development backlog

The topic was posted 10 hours ago...

**Reply by poteitouweb3** [reply]
But the network problem is already extending into its second week. The approach is recent but the problem is not.

**Comment by adamhannigan** [comment]
Thanks @poteitouweb3 for raising this - it definitely helps us prioritise.

We are working on this right now 👍

**Reply by poteitouweb3** [reply]
good to hear that! <3

**Reply by IvanSkazkin** [reply]
> Спасибо@poteitouweb3для поднятия этого - это определенно помогает нам расставить приоритеты.
> 
> Мы работаем над этим прямо сейчас👍

Thanks for your reply. Looking forward to this feature

**Comment by deathmbah666** [comment]
this game need sync to feed my chicken XD , so its must make attention about gas fee, all of us love low gas fee!!! 
#all_hail_GOBLIN!!! #justice_for_gnome

**Comment by szulkarn** [comment]
hi dev. maybe you can consider Redlight Chain. they're totally GASLESS not 0.00000000001 but 0 (zero) .
literally zero.
maybe you can pop in to their discord
https://discord.gg/redlightfinance
and contact Alex 8Bit the CTO of the chain.
maybe you guys can work something out. it would definitely makes the game so much stress free.
 thank you

**Comment by FernandoAvellar** [comment]
Hi guys! Please do the best to fix this problem as soon as possible. This issue is frustrating the game experience. Im my opinion the idea to pay USD 1 to get 10 tickets is the best solution to keep the game experience in high level.
